clipper_admin/docs/conf.py

```diff
@@ -42,13 +42,13 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath('..'))
-import recommonmark
-from recommonmark.parser import CommonMarkParser
-from recommonmark.transform import AutoStructify
-
-source_parsers = {'.md': CommonMarkParser}
+# import recommonmark
```

Comment:
Given that this is just the config to build the docs, I'd actually rather leave that in if we want to add markdown support back, rather than having to go look it up again in the future.